task/task.go

```diff
@@ -0,0 +1,19 @@
+package task
+
+import "encoding/json"
+
+type Task struct {
+	Id        int
+	JobId     int
+	Algorithm string
+	Salt      string
+	Digest    string
+	CharSet   string
+	Length    int
+	Start     string
+	End       string
```

Comment:
I was thinking whether `End` is suitable here. `Start` and like a number of rounds to run might be better, otherwise in the brute force loop the slave have to check whether it has reached the end on every iteration.